devicetypes/smartthings/zigbee-dimmer.src/zigbee-dimmer.groovy

```diff
@@ -113,6 +113,12 @@ def refresh() {
     zigbee.onOffRefresh() + zigbee.levelRefresh()
 }
 
+def installed() {
+    if ((device.getDataValue("manufacturer") == "MRVL") && (device.getDataValue("model") == "MZ100")) {
```

Comment:
Does installed get called every time the DTH is changed? Assuming that's the case it might be safer to only create the event if the current value if 0.